edb/schema/constraints.py

```diff
@@ -517,6 +512,43 @@ def get_ddl_identity_fields(
         else:
             return id_fields
 
+    @classmethod
+    def _ignorenames_from_ast(
+        cls,
+        schema: s_schema.Schema,
+        astnode: qlast.DDLOperation,
+        context: sd.CommandContext,
+    ) -> Set[str]:
+        ignorenames = super(ConstraintCommand, cls)._ignorenames_from_ast(
+            schema, astnode, context
+        )
+        # Set up the constraint parameters as part of names to be
+        # ignored in expression normalization.
+        if isinstance(astnode, qlast.CreateConstraint):
+            ignorenames |= {param.name for param in astnode.params}
+        elif isinstance(astnode, qlast.AlterConstraint):
+            # ALTER ABSTRACT CONSTRAINT doesn't repeat the params,
+            # but we can get them from the schema.
+            objref = astnode.name
+
+            # Merge the context modaliases and the command modaliases
+            # to correctly infer the module where the constraint shoud
+            # be.
+            modaliases = dict(context.modaliases)
+            modaliases.update(
+                cls._modaliases_from_ast(schema, astnode, context))
+            module = modaliases.get(objref.module, objref.module)
```

Comment:
This isn't necessary. `Schema.get()` takes `modaliases` as argument.

Reply:
That's right, I've passed the modaliases directly to `get`.